network/wsNetwork.go

```diff
@@ -1731,6 +1735,41 @@ func (wn *WebsocketNetwork) sendPeerConnectionsTelemetryStatus() {
 	wn.log.EventWithDetails(telemetryspec.Network, telemetryspec.PeerConnectionsEvent, connectionDetails)
 }
 
+// function to send periodic connected peer metrics
+// the peer information is stored as the metric's labels
+func (wn *WebsocketNetwork) sendPeerConnectionsMetricStatus() {
+	ticker := time.NewTicker(10 * time.Second)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-ticker.C:
+			outPeerMap, inPeerMap := wn.updateConnectedPeersLabelMaps()
+			incomingPeerAddr.Set(1, inPeerMap)
```

Comment:
So now a busy peer with 1000 incoming connections has 1000 lines of /metrics output instead of the ~40 it has now? Are we sure we really want this?

Reply:
I'm not sure that we want this solution as is.. but I think that we should hold off until @grakshith has a working end-to-end solution..
Once we have it, he will be able to remove the code-portions that aren't really needed.

Reply:
The connected peers are already being sent as telemetry events. But for some reason, I couldn't see them on the ephemeral telemetry elasticsearch. I added these metrics for now, to help build the monitoring system, and as @tsachiherman said, I will remove the bits not needed once I have the complete solution. I will convert the PR to draft.